### DISCUSSION TITLE
Fractional minute

### DISCUSSION THREAD

**Discussion by manozzo** [moment/moment#6431] [Q&A] [answered]
Hi everyone.

I want to use moment in my stopwatch project, but here in Brazil most of automotive companies measures the time in decimal minute.

I would like a fractional minute format, for example 45 seconds.
Instead show me 00:42 formatted 'mm:ss', I would like to see 0.75min like 'mm.(fracMinute)'.

Another option is format to unit 'cmin' = hundredth of minute. Example: 0.75min = 75cmin

I need to format always in minutes, decimal minutes.

Thank you all,

Kind regards,

**Comment by mattjohnsonpint** [comment, marked as answer]
Thanks for the question! I'd calculate this as a duration: `moment.duration(45, 'seconds').asMinutes()` returns `0.75`. Multiplying that by `100` gives `75` centiminutes, and JavaScript's `toFixed()` can control display precision.
